src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -312,9 +312,13 @@ public async Task<ActionResult<bool>> ProcessPendingUpdatesAsync()
 
             (InProgressPullRequest pr, bool canUpdate) = await SynchronizeInProgressPullRequestAsync();
 
+            var subscriptionIds = updates.Count > 1
+                ? "subscriptions " + string.Join(", ",updates.Select(u => u.SubscriptionId).Distinct())
+                : "subscription " + updates[0].SubscriptionId;
+
             if (pr != null && !canUpdate)
             {
-                _logger.LogInformation($"PR {pr?.Url} cannot be updated.");
+                _logger.LogInformation("PR {url} for {subscriptions} cannot be updated", pr.Url, subscriptionIds);
```

Comment:
I only put it in this method because it's the most high level point of the update flow. The idea was that if you search for the sub ID, you will immediately see the result of the trigger. E.g. you'd see all times this subscription was getting updated.  
There should be just one log statement per subscription here.

If we did what you propose, we'd have to repeat a search for every time this subscription was triggered. I am not sure that is better?